lib/core/presentation/theming/themes/light_theme.dart

```diff
@@ -63,7 +73,7 @@ ThemeData whiteTheme(BuildContext context, BoxConstraints cts) {
       ),
       titleSmall: style.titleSmall?.copyWith(
         fontSize: cts.maxWidth * 0.04,
-        color: _AppColors.onSurfaceW,
+        color: _AppColors.greyLight,
```

Comment:
- The app background color should be some shade of white `#F7F7F7` not the current greyish color
- The currency is missing the dollar sign as pointed out in the image
-  The like icon should be right-aligned not centered
- The New Category button should be black, not blue
- Is the pie chart currently able to represent different colors for different categories?

![pr-rv](https://user-images.githubusercontent.com/17765231/222711783-575291ae-9cba-40b4-add9-9b84d94779ee.jpeg)